corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py

```diff
@@ -164,10 +165,25 @@ def ensure_plans(dry_run, verbose, for_tests, apps):
                         logger.info("Creating Software Plan: %s" % software_plan.name)
 
                     software_plan_version.plan = software_plan
-                    software_plan_version.save()
+
+                    # must save before assigning many-to-many relationship
+                    if hasattr(SoftwarePlanVersion, 'product_rates'):
+                        software_plan_version.save()
+
                     for product_rate in product_rates:
                         product_rate.save()
-                        software_plan_version.product_rates.add(product_rate)
+                        if hasattr(SoftwarePlanVersion, 'product_rates'):
+                            software_plan_version.product_rates.add(product_rate)
+                        elif hasattr(SoftwarePlanVersion, 'product_rate'):
+                            assert len(product_rates) == 1
+                            software_plan_version.product_rate = product_rate
+                        else:
+                            raise AccountingError('SoftwarePlanVersion does not have product_rate or product_rates field')
```

Comment:
I don't have a specific idea of how the ```else``` block would be reached.  This is more of a "if a breaking change is introduced, fail loudly" situation